graph/resolver/schema.resolvers.go

```diff
@@ -87,20 +87,18 @@ func (r *mutationResolver) PostImportedContent(ctx context.Context, discussionID
 		return nil, fmt.Errorf("unauthorized")
 	}
 
-	participant, err := r.DAOManager.GetParticipantByID(ctx, participantID)
+	// Get concierge participant
+	resp, err := r.DAOManager.GetParticipantsByDiscussionIDUserID(ctx, discussionID, model.ConciergeUser)
 	if err != nil {
+		logrus.WithError(err).Error("failed to fetch concierge participant")
 		return nil, err
-	} else if participant == nil {
-		return nil, fmt.Errorf("Could not find Participant with ID %s", participantID)
 	}
-
-	// Verify that the posting participant belongs to the logged-in user
-	if *participant.UserID != authedUser.UserID {
-		return nil, fmt.Errorf("Unauthorized")
+	if resp.NonAnon == nil {
+		return nil, fmt.Errorf("discussion is missing a concierge participant")
```

Comment:
Given this is at the resolver level we should probably log an error (if it's not done lower down) and then obscure this message?

Reply:
updated